docs/src/pages/components/skeleton.mdx

```diff
@@ -112,6 +112,14 @@ You might just want to give the Skeleton specific dimensions to make it take the
 ## API
 
 <ApiTable data={[
+    {
+        property: 'children',
+        description: `In this particular instance, the <pre>children</pre> of <pre>Skeleton</pre>
+            are very important as they determine the size/shape. Note that using
+            <pre>textLength</pre> will override the size of any child element, or the length of any
+            text.`,
+        type: 'any',
```

Comment:
Perhaps `string` | `any`. Maybe I should make the code that way too.

Reply:
Also, should probably mention (redundantly) how its important (i.e. text vs nontext)